tests/programs/custom/zcmt_test/zcmt_test.c

```diff
@@ -0,0 +1,1744 @@
+//
```

Comment:
When accepting this test into the cv32e40s repo it is expected to do testing.
Ideally we should test the test for cv32e40x before merging it over. Do we know if it is tested for X as well?

Reply:
It is not tested. (Likely, it also fails.)

Although the `zcmt_test` test's yaml description says _"zcmt directed test"_, there are further clues in the `full` regression yaml: _"user-mode, mstateen0 directed test"_.
(The rejection diff shows that this is not included in the "full" regression file for the X.)

So the test seems user mode specific, not X-relevant.
Further, it doesn't seem X-relevant because there is no `has_umode_configured()` in that C file.

I don't think it belongs in X.
But I also think we should minimize the S/X diff.
Hence, I think we should just merge it over and let it lay there.

Reply:
Okei, I see, I though maybe the test was "parameterized out", so maybe it would run and only x relevant code was executed, but yeah, it is probably not that important anyways. Okei, let merge it then. RESOLVED